src/settings/settings.ts

```diff
@@ -52,5 +54,16 @@ export class ContactsSettingTab extends PluginSettingTab {
           this.plugin.settings.template = value as Template;
           await this.plugin.saveSettings();
         }));
+
+    new Setting(containerEl)
+      .setName('Default hashtag')
+      .setDesc('Hashtag to be used for every contact created')
+      .addDropdown(text => text
```

Comment:
There are no options in dropdown
<img width="777" alt="image" src="https://github.com/vbeskrovnov/obsidian-contacts/assets/9114994/abb14b80-79ca-4a02-b757-8978cddfe245">

Could you please use text field instead?

Reply:
Thank you, fixed!